src/iso15118/d20/state/dc_charge_parameter_discovery.cpp

```diff
@@ -42,6 +42,81 @@ template <> void convert(BPT_DC_ModeRes& out, const d20::DcTransferLimits& in) {
     }
 }
 
+bool handle_compatibility_check(const d20::DcTransferLimits& evse_dc_limits, const session::feedback::DcMaximumLimits ev_max_limits) {
+    // In IEC 61851-23-3 a compatibility check is required
+    // There are voltage, current and power values indexed with CPD and RATED.
+    // The CPD is the communicated value in DC_ChargeParameterDiscoveryRes and RATED the value coming from
+    // the EVerest environment (power supply, grid), here we assume that both are the same
```

Comment:
For readability I think you can remove cpd and rated. I think it is obvious which parameter comes from where (evse or ev)

Reply:
After reading CC.5.6 more carefully, I now understand where the CPD and Rated names come from. Nevertheless, they make less sense to me in code.